### ISSUE TITLE
Updating the global CLI version to v6.1.1 from v1.7.4 breaks "ng serve" and "ng --version"; perhaps others, too 

### ISSUE THREAD

**Issue by Nerketur** [angular/angular-cli#11713]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [ x ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ x ] version
- [ ] doc
```

### Versions

OS: Windows 8.1

terminal: Windows PowerShell

node --version:
`v8.11.3`

npm --version:
`5.6.0`

ng --version:
```
Your global Angular CLI version (6.0.8) is greater than your local
version (1.7.4). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".

    _                      _                 ____ _     ___
   / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
  / ? \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
 / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
               |___/

Angular CLI: 1.7.4
Node: 8.11.3
OS: win32 x64
Angular: 5.2.11
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cli: 1.7.4
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.2
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.5.3
webpack-bundle-analyzer: 2.13.1
webpack: 3.11.0
```
### Repro steps

1. create a project using Angular CLI v1.7.4 `ng new proj-name`, or similar 
2. update global angular CLI version to 6.1.1
    `npm uninstall -g @angular/cli`
    `npm install -g @angular/cli` (or add `@6.1.1` if needed)
3. try to run `ng serve --aot`

### The log given by the failure

```
Your global Angular CLI version (6.1.1) is greater than your local
version (1.7.4). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".
Cannot read property 'write' of undefined
TypeError: Cannot read property 'write' of undefined
    at UI.write (C:\path\to\angular-project\node_modules\@angular\cli\ember-cli\lib\ui\index.js:56:23)
    at UI.writeLine (C:\path\to\angular-project\node_modules\@angular\cli\ember-cli\lib\ui\index.js:69:8)
    at Class.run (C:\path\to\angular-project\node_modules\@angular\cli\tasks\serve.js:237:12)
    at check_port_1.checkPort.then.port (C:\path\to\angular-project\node_modules\@angular\cli\commands\serve.js:123:26)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

### Desired functionality

I would like to be able to run a `@angular/cli@1.7.4` project with the latest version of angular, so I don't have to use the latest angular on extremely old browsers, and can update as needed, while still being able to make new projects with the latest.

### Mention any other details that might be useful

This is related to issue #11661, but that was closed because of inability to see the local version.  I am unable to get `ng --version` to work on a 1.7.4 project with CLI v6.1.1, but using the tip for downgrading, everything seems to work on CLI v6.0.8.  that is why the `ng --version` info says the global version is v6.0.8

deleting the node_modules folder and running `npm install` to redownload everything does not help in v6.1.1

as a workaround, CLI v6.0.8 can be used with no issues

**Comment by wattachai** [comment]
I ran into the same issue and confirmed that the CLI v6.0.8 is a valid workaround.

**Comment by hansl** [comment]
Fixed by #11690
